src/components/button/_button.scss

```diff
@@ -241,14 +232,11 @@
   }
 
   .#{$prefix}--btn--ghost {
-    @include button-theme--x(
-      transparent,
-      transparent,
-      $interactive-03,
-      $ibm-colors__gray-10,
-      $interactive-03,
-      $ibm-colors__blue-70
-    );
+    @include button-theme--x(transparent, transparent, $interactive-01, $hover-ui, $interactive-01, $active-ui);
+    display: inline-flex;
+    align-items: center;
+    min-width: initial;
```

Comment:
`initial` seems not supported in IE11: https://developer.mozilla.org/en-US/docs/Web/CSS/initial

Reply:
I replaced the `initial` value with `auto`